scripts/integration.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/sh
```

Comment:
I'd be ok with doing it in Python, I just chose bash since I'm more familiar with it. If you want we can use this one for now and replace it once yours is ready.